boa/src/syntax/parser/mod.rs

```diff
@@ -1728,4 +1687,52 @@ impl<'a> Parser<'a> {
             Some(pos),
         ))
     }
+
+    /// <https://tc39.es/ecma262/#prod-MethodDefinition>
+    fn read_property_method_definition(
+        &mut self,
+        identifier: &str,
+    ) -> Result<PropertyDefinition, ParseError> {
+        let (methodkind, prop_name, params) = match identifier {
+            "get" | "set" => {
+                let prop_name = self.get_object_property_name()?;
+                self.expect(
+                    TokenKind::Punctuator(Punctuator::OpenParen),
+                    Some("property method definition"),
+                )?;
+                let params = self.read_formal_parameters()?;
+                if identifier == "get" {
+                    if !params.is_empty() {
```

Comment:
What is the best way to handle errors here? Getters should have no arguments, while setters take only one.

Reply:
If in doubt try putting an argument in a getter on a typical js environment and see what sort of error you get

Reply:
Ok yep, I updated the PR.